packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -113,7 +113,7 @@ const App = (props) => {
     const {children} = props
     const {data: categoriesTree} = useLazyLoadCategories()
     const categories = flatten(categoriesTree || {}, 'categories')
-
+    const {token} = useAccessToken()
```

Comment:
```suggestion
    const {getTokenWhenReady} = useAccessToken()
```
How about using `getTokenWhenReady` instead? Because the token can be null initially.